code/modules/clothing/rogueclothes/cloaks.dm

```diff
@@ -951,7 +951,6 @@
 	color = null
 	icon_state = "battlenun"
 	item_state = "battlenun"
-	allowed_sex = list(FEMALE)
```

Comment:
the battlenun clothing does not have a male sprite that fits

Reply:
Aye smited it, thanks for the warning